src/screens/Settings/index.tsx

```diff
@@ -227,7 +227,11 @@ function Settings() {
       setAddDesktopShortcuts(config.addDesktopShortcuts || false)
       setAddGamesToStartMenu(config.addStartMenuShortcuts || false)
       setCustomWinePaths(config.customWinePaths || [])
-      setCheckForUpdatesOnStartup(config.checkForUpdatesOnStartup || true)
+      setCheckForUpdatesOnStartup(
```

Comment:
I believe this doesnt fix the issue because this check does the same as before. The issue is on line:
https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/blob/5d72a3860e427ba68267b7d5906b11ba0c3d27f1/src/screens/Settings/index.tsx#L94

Reply:
I will check again. I mean for me it fixed it.
For my Understanding:
undefined || true = true
false || true = true
-> Actually if the value is false you wanna set it to false and not true!

So maybe we need to fix both places ?

Reply:
I tested the option alot. set it, close the app and open again. Stores the value now. Before it was set always to true because of that ||

Reply:
In my eyes it is pretty simple logical stuff there. Doing || with a fixed boolean value is never good!

Reply:
Tested System Tray options and everything seems fine after the removing